docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -0,0 +1,502 @@
+# Migration from v4 to v5
```

Comment:
It looks like this file shouldn't be on the master branch, but only the next branch.